sopel/modules/announce.py

```diff
@@ -7,18 +7,16 @@
 """
 from __future__ import unicode_literals, absolute_import, print_function, division
 
-from sopel.module import commands, example
+from sopel.module import commands, example, require_admin
 
 
 @commands('announce')
 @example('.announce Some important message here')
+@require_admin('Sorry, I can\'t let you do that', reply=True)
```

Comment:
While you're in here: should this message be updated to say why "`[the bot] can't let you do that`"? i.e., tell user that only admins can do that...

Thoughts?

Reply:
Since I've 0 opinion bout that, I call @dgw like the coward I am. 😛

Reply:
Update it if you want to, don't if you don't. The message has been that way for almost six years. :stuck_out_tongue:

Reply:
If you're doing "One change at a time", what's up with adding a new parameter to `sopel.module.require_admin` in a PR titled "use \@require_admin instead of if trigger.admin"? 😁

Reply:
@dgw I know. Optional parameter + same behavior for the end-user as before is what I went for. :sunglasses:

Reply:
I'd kind of prefer that kind of change in its own PR, or at least its own commit… How much trouble would that be? Would you be mad? :fearful:

Reply:
I'll try that in a second commit then, because I can not replace `if not trigger.admin` check without this change, because some command want to reply something.

Reply:
Rebased & fix.

Reply:
Cool! Though ipython is updated in the commit after adding the new arg but doesn't use it?

Reply:
Oh, I thought it was but... no, it doesn't. Why did I do that? I think I understand why you were confused the first time, because I sure am confused right now.

Reply:
Oh, ok, I found out:

* first commit: `@require_admin` without message
* second commit: add parameter
* third commit: `@require_admin` with message (and one with reply)

So it checks out. I think.

Reply:
That's what I expected, but the "with reply" commit is where ipython gets updated. It doesn't use reply.